coax/__init__.py

```diff
@@ -57,6 +57,7 @@
     # classes and functions
     'V',
     'Q',
+    'QuantileQ',
```

Comment:
Do we still need this after refactor?

Reply:
I missed the reference but removed it in a subsequent commit.